service/src/LevelZeroIOGroup.cpp

```diff
@@ -115,6 +115,22 @@ namespace geopm
                                   },
                                   1e6
                                   }},
+                              {M_NAME_PREFIX + "GPU_CORE_THROTTLE_REASONS", {
+                                  "GPU Compute/Core domain throttle reasons.  See oneAPI Level Zero Sysman Spec for decoding",
+                                  GEOPM_DOMAIN_GPU_CHIP,
+                                  Agg::average,
```

Comment:
A bitwise OR makes more sense here for aggregation as it would represent all the throttling across the device compute domains